Sources/UID2/Properties/sdk_properties.plist

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>UID2Version</key>
+    <string>0.1.0</string>
```

Comment:
Nice.

Not for now, but should we document the release process of the SDKs. We need to make sure that this version is bumped during that process. I guess we will also end up with a `RELEASENOTES.md` file or equivalent too.

Reply:
Yep, we absolutely need to document this.  Determining the release process is already on the TODO list --> 

https://atlassian.thetradedesk.com/jira/browse/UID2-847